internal/core/src/segcore/Utils.h

```diff
@@ -110,11 +110,15 @@ MergeDataArray(std::vector<MergeBase>& merge_bases,
 
 template <bool is_sealed>
 std::shared_ptr<DeletedRecord::TmpBitmap>
-get_deleted_bitmap(int64_t del_barrier,
-                   int64_t insert_barrier,
-                   DeletedRecord& delete_record,
-                   const InsertRecord<is_sealed>& insert_record,
-                   Timestamp query_timestamp) {
+get_deleted_bitmap(
```

Comment:
how about move this to member function of SegmentInternalInterface.

Reply:
It becomes more complex to place it in the `SegmentInternalInterface` class because the base class `SegmentInternalInterface` does not contain any members, such as `insert_record_`, `delete_record_`, or `fields`. These member variables are not necessarily shared, for example, `fields`. In the case of a sorted sealed segment, the primary key is queried through the original data in `fields`. So even if the function is placed in `SegmentInternalInterface`, additional function parameters would still be needed to query the primary key through `fields`.